source/standards/accounts-with-third-parties.html.md.erb

```diff
@@ -0,0 +1,146 @@
+---
+title: Managing accounts with third party services
+expires: 2018-04-01
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+When developing a service, your team will often require the use of
+third-party services.  To access these services, you will need some
+sort of user account.  This page provides guidance on how to manage
+those accounts.
+
+## Terms
+
+In this page we distinguish between an account and an individual:
+
+- An *account* means an account with an identifier such as a username or
+  email address
+- An *individual* means a human being
+- A set of *credentials* means some sort of secret used to gain access
+  to a user account (for example: a password, a secret access key)
+
+## Goals and principles
+
+When managing credentials we have a few goals:
+
+- Ensure that the right people have access
+- Ensure that the wrong people do not have access
+  - Particularly: ensure when people leave the organization, their
+    access is revoked or expired
+- Identify who did what and when (audit)
+  - This goal has varying importance depending on context
+  - Technical limitations may prevent the goal from being achieved, or
+    only provide partial audit
+
+To meet these goals, we have some principles:
+
+- Prefer to minimise the number of secrets that an individual needs to
+  keep
+  - This helps with revocation because there are fewer sets of
+    credentials to revoke when someone leaves
+  - Passwords are particularly problematic secrets. Other forms of
+    secrets (eg secret access keys or OAuth tokens) are more easily
+    revoked and recycled.
+- No individual should ever have to use credentials associated with
+  another individual
+  - This would be counter to the audit goal
+- There should be no need to share credentials between teams
+  - Again, this is problematic for audit
+  - This is also problematic for knowing when credentials should be
+    revoked
+  - Technical limitations may mean this is unavoidable sometimes
+
+## Creating accounts for shared third-party services
+
+The recommendations here depend on whether the service supports
+organisation or team access natively.
+
+### Services that support organisation accounts
```

Comment:
"account" in "organisation account" here doesn't seem right. An "account" is defined earlier that an account has an identifier and log-in credentials, and for GitHub no-one has credentials for "alphagov" itself. GitHub only really talks about "organization accounts" in terms of billing. At HMRC use "organisation account" for a log-in to a business. Google talks about "organizational accounts" as user's work log-in. I believe you're instead talking authz applying to a *group* of user accounts. You might instead call it 'team-level permissions' or 'teams of users for permissions'.

Reply:
I'm looking for a word or phrase that describes what `alphagov` is on github.  Do you have a better alternative than "organisation account"?

Reply:
Does 'organisations' suffice by itself? ("Services that support organisations")

Reply:
@soupdragon99 to address

Reply:
After some head scratching, I couldn't come up with a suitable alternative. Basically, there's no definitive usage by different companies\vendors in how they use 'organisation' and 'accounts'. To avoid a lengthy title, I think in this context we could leave it, as it is but add something to the intro paragraph to clarify. Although tbh I think it's pretty clear anyway given the overall guidance.

Reply:
Agreed, it's fine for a title, but please do add what you mean by "organisation account" or the advantage you perceive. I think you're thinking it is good because by putting a repo in alphagov then some senior GDS people get given admin permissions automatically. Alternatively you might consider saying it's good to use a centralized AWS account so that fewer people have to deal with billing and there can be some centralized security monitoring, although admittedly these concerns are not part of your stated goals.